MinPlatformPkg/Test/Library/TestPointCheckLib/StandaloneMmCheckSmrr.c

```diff
@@ -0,0 +1,135 @@
+/** @file
```

Comment:
This file seems to be the same as smm one.

Reply:
Consolidated.